test/controllers/DepartureTestMessagesControllerSpec.scala

```diff
@@ -121,7 +121,7 @@ class DepartureTestMessagesControllerSpec extends SpecBase with ScalaCheckProper
       val invalidRequest = Json.parse(
         """{
           |     "message": {
-          |         "messageType": "IE028"
+          |         "messageType": "IE005"
```

Comment:
Is IE028 not good enough any more?

Reply:
IE028 is now considered a valid message 😄